src/main/drivers/bus.h

```diff
@@ -18,7 +18,26 @@
 #pragma once
 
 #include "platform.h"
+#include "drivers/io_types.h"
 
 #ifdef TARGET_BUS_INIT
 void targetBusInit(void);
 #endif
+
+typedef enum {
+    BUSTYPE_NONE = 0,
+    BUSTYPE_I2C,
+    BUSTYPE_SPI,
+} busType_e;
+
+#define BUSTYPE_NONE_STR "NONE"
```

Comment:
This indirection is a bit of overkill. If you want to keep cli in sync easily, use:
```
#define BUSTYPE_LOOKUP_TAB "NONE", "I2C", "SPI"